src/draw/nema_gfx/lv_draw_nema_gfx_img.c

```diff
@@ -170,13 +172,25 @@ static void _draw_nema_gfx_img(lv_draw_unit_t * draw_unit, const lv_draw_image_d
         blending_mode |= NEMA_BLOP_MODULATE_A;
     }
 
-    if(dsc->bitmap_mask_src != NULL) {
-        blending_mode |= NEMA_BLOP_STENCIL_TXTY;
-        const lv_image_dsc_t * mask = dsc->bitmap_mask_src;
-        const void * mask_buf = mask->data;
-        nema_bind_tex(NEMA_TEX3, (uintptr_t)NEMA_VIRT2PHYS(mask_buf), mask->header.w, mask->header.h,
-                      lv_nemagfx_mask_cf_to_nema(mask->header.cf),
-                      mask->header.stride, NEMA_FILTER_BL);
+    if(!has_transform && masked && !recolor) {
+        if(dsc->bitmap_mask_src->header.cf == LV_COLOR_FORMAT_A8 || LV_COLOR_FORMAT_L8) {
+            blending_mode |= NEMA_BLOP_STENCIL_TXTY;
+            const lv_image_dsc_t * mask = dsc->bitmap_mask_src;
+            const void * mask_buf = mask->data;
+
+            lv_area_t * image_area, mask_area;
```

Comment:
`image_area` should be const since `coords`, a const pointer, is being assigned to it.

Prefer one line per declaration.

```suggestion
            const lv_area_t * image_area,
            lv_area_t mask_area;
```